src/kava.js

```diff
@@ -746,20 +766,30 @@ class Kava extends BasePlugin {
     return (Date.now() - time) / 1000.0;
   }
 
-  _isDocumentHidden(): boolean {
-    let hidden = '';
+  _updateTabModeinModel(): void {
+    this._model.updateModel({
+      // $FlowFixMe
+      tabMode: document[this._hiddenAttr] ? TabMode.TAB_NOT_FOCUSED : TabMode.TAB_FOCUSED
+    });
+  }
+
+  _initTabMode(): void {
     if (typeof document.hidden !== 'undefined') {
       // Opera 12.10 and Firefox 18 and later support
-      hidden = 'hidden';
+      this._hiddenAttr = 'hidden';
```

Comment:
move to `const` and send to `_updateTabModeinModel` as a parameter

Reply:
done. but as let and not const as it is assigned not upon declaration and fails in eslint